ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/impl/inmemory/InMemoryUserManagementStore.scala

```diff
@@ -118,7 +118,8 @@ class InMemoryUserManagementStore(createAdmin: Boolean = true) extends UserManag
 object InMemoryUserManagementStore {
 
   private val AdminUser = UserInfo(
-    user = User(Ref.UserId.assertFromString("participant_admin"), None),
+    user =
+      User(Ref.UserId.assertFromString(UserManagementStore.DefaultParticipantAdminUserId), None),
```

Comment:
I think a unit test for this would be nice but I'm not 100% familiar with the user management stuff.

Reply:
There's already a unit test for InMemoryUserManagementStore